zeroex/order.go

```diff
@@ -174,6 +215,19 @@ func (s *SignedOrder) ConvertToOrderWithoutExchangeAddress() wrappers.OrderWitho
 	return orderWithoutExchangeAddress
 }
 
+// ConvertToSignedOrder converts an order to a signed 0x order
+func ConvertToSignedOrder(order *Order, rpcClient *rpc.Client) (*SignedOrder, error) {
```

Comment:
I don't like that there are two exported identifiers for signing an order which do relatively similar things. How about making `Order.ECSign` unexported and renaming `ConvertToSignedOrder` to `SignOrder`? Then there's only one exported function called `SignOrder` with a clear purpose.